src/client/components/Swipe/Swipe.jsx

```diff
@@ -107,9 +108,34 @@ class Swipe extends Component {
     };
   }
 
+  getPlaceData(placeId) {
+    const instance = this;
+    axios
```

Comment:
Can we abstract out the requests into a separate class? This way we can keep it clean and simple.

What I mean is that we create classes for interfaces with the API such as PlaceDataInterface which contains methods such as getPlaceData() and other calls. This class will be a subclass of a general DataInterface class that implements the first level request methods. 

This way we can keep all the API utils bundled together and avoid bugs during implementation. It will be easy to have auth, utils and more together.

Reply:
Sounds good, let me add that to trello.

Reply:
But it will be quite a few changes so I think it should be a whole other PR. Is that okay?

Reply:
Sounds good to me